pkgs/applications/networking/instant-messengers/signal-desktop/default.nix

```diff
@@ -2,5 +2,7 @@
   signal-desktop = if hostPlatform.system == "aarch64-linux"
     then callPackage ./signal-desktop-aarch64.nix { }
     else callPackage ./signal-desktop.nix { };
-  signal-desktop-beta = callPackage ./signal-desktop-beta.nix{ };
+  signal-desktop-beta = if hostPlatform.system == "x86_64-linux"
+    then callPackage ./signal-desktop-beta.nix { }
+    else { };
```

Comment:
is returning empty set a good idea ?

maybe something like ( didn't test )

`(callPackage ./signal-desktop-beta.nix { }).overrideAttrs old: { meta = old.meta // {broken = true;} }`

Reply:
Good idea. Implemented.

Reply:
Please use `targetPlatform.system` over `hostPlatform.system` because `targetPlatform.system` is the platform for which you are building, while `hostPlatform.system` is the platform where the build is happening.